Detectors/ITSMFT/ITS/workflow/src/DCSParserSpec.cxx

```diff
@@ -482,8 +482,8 @@ void ITSDCSParser::pushToCCDB(ProcessingContext& pc)
     tstart = o2::ccdb::getCurrentTimestamp();
     tend = tstart + 365L * 24 * 3600 * 1000;
   } else {
-    tstart = std::stol(headers["SOR"]);
-    tend = std::stol(headers["EOR"]);
+    tstart = std::stol(headers["STF"]);
+    tend = std::stol(headers["ETF"]);
```

Comment:
@iravasen Actually, it is better to use instead of https://github.com/AliceO2Group/AliceO2/pull/13165/files#diff-518e73d130cc9d7a34433d6e4320e6fbdbe7f399b4c9df38d32bdb7003235668L471-L486

```
auto ts = o2::ccdb::BasicCCDBManager::instance().getRunDuration(551363);
long tstart = ts.first, tend = ts.second;
```
This will profit from the fallback set in the getRunDuration in case the ETF or STF are not there or invalid.

Reply:
Thanks @shahor02 , indeed a much better solution. I've implemented it.